frontend/src/metabase/lib/settings.js

```diff
@@ -102,16 +99,24 @@ class Settings {
     return this.get("setup-token") != null;
   }
 
-  ssoEnabled() {
-    return this.get("google-auth-client-id") != null;
+  hideEmbedBranding() {
+    return this.get("hide-embed-branding?");
   }
 
   ldapEnabled() {
     return this.get("ldap-configured?");
   }
 
-  hideEmbedBranding() {
-    return this.get("hide-embed-branding?");
+  searchTypeaheadEnabled() {
+    return this.get("search-typeahead-enabled");
+  }
+
+  ssoEnabled() {
+    return this.get("google-auth-client-id") != null;
+  }
+
+  trackingEnabled() {
+    return this.get("anon-tracking-enabled") || false;
```

Comment:
diff noise since I did some alphabetizing and renamed `isTrackingEnabled` to `trackingEnabled` to make it more consistent with the rest of them